lib/project_types/extension/models/specification_handlers/checkout_ui_extension.rb

```diff
@@ -62,6 +62,15 @@ def localization(context)
           end
         end
 
+        def read_locale_file(filename)
+          Base64.strict_encode64(File.read(filename, mode: "rt", encoding: "UTF-8").strip.sub("\xEF\xBB\xBF", ""))
```

Comment:
Could you add a comment that you're replacing the byte order mark? It may not be obvious.